cms/djangoapps/api/v1/views/course_runs.py

```diff
@@ -27,8 +27,8 @@ def get_object(self):
         lookup_url_kwarg = self.lookup_url_kwarg or self.lookup_field
 
         assert lookup_url_kwarg in self.kwargs, (
-            'Expected view %s to be called with a URL keyword argument '
-            'named "%s". Fix your URL conf, or set the `.lookup_field` '
+            u'Expected view %s to be called with a URL keyword argument '
```

Comment:
@cpennington i have a couple places where multi-line strings are each prefixed with u
i'm assuming that the correct thing to do is just the prefix on the first line (and i've been fixing these) but want to double check before i fix the rest